docs/source/json-api/index.rst

```diff
@@ -1053,6 +1055,54 @@ Streaming API
 Two subprotocols must be passed with every request, as described in
 `Passing token with WebSockets <#passing-token-with-websockets>`__.
 
+JavaScript/Node.js example demonstrating how to establish Streaming API connection:
+
+.. code-block:: javascript
+
+    const WebSocket = require("ws")
+
+    console.log("Starting")
+
+    const tokenPrefix = "jwt.token."
+    const wsProtocol = "daml.ws.auth"
+    const jwt =
+      "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJodHRwczovL2RhbWwuY29tL2xlZGdlci1hcGkiOnsibGVkZ2VySWQiOiJNeUxlZGdlciIsImFwcGxpY2F0aW9uSWQiOiJmb29iYXIiLCJhY3RBcyI6WyJBbGljZSJdfX0.VdDI96mw5hrfM5ZNxLyetSVwcD7XtLT4dIdHIOa9lcU"
+    const subprotocol = `${tokenPrefix}${jwt},${wsProtocol}`
+
+    const ws = new WebSocket("ws://localhost:7575/v1/stream/query", [subprotocol])
+
+    ws.on("open", function open() {
+      ws.send(`{"templateIds": ["Iou:Iou"]}`)
```

Comment:
I think you would typically write the JS object as an object and then encode it to JSON, yes?

Reply:
this is Javascript, no types who cares :) but yeah `JSON.stringify` should do the trick. I will change it.

Reply:
```
    ws.on("open", function open() {
      ws.send(JSON.stringify({templateIds: ["Iou:Iou"]}))
    })
```

Reply:
For sure :) you can at least catch dumb syntax typos like missing `}` or `"` this way.